src/style/art-gallery.scss

```
@@ -0,0 +1,108 @@
+@import 'var.scss';
+.Art-gallery{
```

давай называть стили с маленькой буквы в kebab-case. И не забывай про форматирование кода (пробел между названием селектора и {)

```suggestion
.art-gallery {
```